crates/router/src/routes/app.rs

```diff
@@ -221,15 +221,19 @@ impl Customers {
         {
             route = route
                 .service(web::resource("").route(web::post().to(customers_create)))
+                .service(
+                    web::resource("/payment_methods")
+                        .route(web::get().to(list_customer_payment_method_api)),
+                )
+                .service(
+                    web::resource("{customer_id}/payment_methods")
```

Comment:
Shouldn't it be starting with `/` here, Is this flow tested

Reply:
yes, but still it was working. have posed ss for the same !